src/desktop/apps/artist/client.tsx

```diff
@@ -13,6 +13,7 @@ buildClientApp({
   routes,
   context: {
     user: sd.CURRENT_USER,
+    ARTIST_INSIGHTS_V2: sd.ARTIST_INSIGHTS_V2_ENABLED && sd.ARTIST_INSIGHTS_V2,
```

Comment:
i don't think you have to do this, all sharify stuff is automatically shared with Reaction.